tools/c7n_mailer/c7n_mailer/slack_delivery.py

```diff
@@ -90,10 +90,16 @@ def get_to_addrs_slack_messages_map(self, sqs_message):
                     resource_list,
                     self.logger, 'slack_template', 'slack_default',
                     self.config['templates_folders'])
-            elif target.startswith('slack://tag/') and 'tags' in resource:
+            elif target.startswith('slack://tag/') and 'Tags' in resource:
                 tag_name = target.split('tag/', 1)[1]
-                result = resource.get('tags', {}).get(tag_name, None)
-                resolved_addrs = result
+                result = next((item for item in resource.get('Tags', [])
```

Comment:
thats kinda of cool, i think thats two new syntaxes i don't normally see (generator expression and default value to next).